src/lib/pmd/PmdEngine.ts

```diff
@@ -104,7 +104,7 @@ abstract class BasePmdEngine extends AbstractRuleEngine {
 				return Severity.HIGH;
 			case 2:
 				return Severity.MODERATE;
-			case 3: 
+			case 3:
```

Comment:
Can we remove this since it's not related to the PR?

Reply:
@jbartolotta-sfdc , done.